src/mixins/filing-template-mixin.ts

```diff
@@ -141,46 +142,30 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
       // make a copy so we don't change original array
       let parties = cloneDeep(this.getOrgPeople)
 
-      // add completing party (client error correction only)
+      // add completing party (client error corrections only)
       if (this.isClientErrorCorrection) {
         parties = this.addCompletingParty(parties)
       }
 
+      // prepare parties
+      parties = isDraft ? parties : this.prepareParties(parties)
+
       // fix schema issues
       parties = this.fixPartySchemaIssues(parties)
 
       filing.correction.parties = parties
     }
 
-    // add in Incorporation Application data
+    // add in data specific to Incorporation Applications
     // *** FUTURE: change this to "if BEN correction"
     if (this.isCorrectedIncorporationApplication) {
-      const nameTranslations = isDraft ? this.getNameTranslations : this.prepareNameTranslations()
-      const shareClasses = isDraft ? this.getShareClasses : this.prepareShareClasses()
-      const parties = isDraft ? this.getOrgPeople : this.prepareParties()
-
-      filing.correction.nameTranslations = nameTranslations
-      filing.correction.shareStructure = { shareClasses }
-
-      // *** FUTURE: save this data in correction object instead
-      // populate the incorporation application object (for the Filer to process)
-      filing.incorporationApplication = {
-        contactPoint: this.getContactPoint,
-        nameRequest: {
-          legalType: this.getEntityType,
-          legalName: this.getNameRequestLegalName,
-          nrNumber: this.getNameRequestNumber
-        },
-        nameTranslations: nameTranslations,
-        offices: this.getOfficeAddresses,
-        parties,
-        shareStructure: { shareClasses }
+      filing.correction.nameTranslations = isDraft ? this.getNameTranslations : this.prepareNameTranslations()
+      filing.correction.shareStructure = {
+        shareClasses: isDraft ? this.getShareClasses : this.prepareShareClasses()
       }
```

Comment:
This IA object is gone! All the fields in here are already saved in the correction object above.